vulcanai2/models/layers.py

```diff
@@ -63,8 +64,17 @@ def _init_bias(self):
         will be assigned to the kernel
         """
         if self.bias_init:
-            nn.init.constant_(self._kernel.bias, self.bias_init)
+            self.bias_init(self._kernel.bias)
 
+def selu_weight_init_(tensor, mean=0):
+    with torch.no_grad():
+        fan_in, _ = nn.init._calculate_fan_in_and_fan_out(tensor)
+        std = math.sqrt(1. / fan_in)
+        return nn.init.normal_(tensor.data, mean, std)
```

Comment:
you don't need to get the `tensor.data`. that is a deprecated tool. just have it be `nn.init.normal_(tensor, mean, std)` the operation is already done in place